birdhouse/components/cowbird/default.env

```diff
@@ -62,6 +70,7 @@ COWBIRD_MONGODB_DATA_DIR='${DATA_PERSIST_ROOT}/mongodb_cowbird_persist'
 DELAYED_EVAL="
   $DELAYED_EVAL
   COWBIRD_MONGODB_DATA_DIR
+  COWBIRD_PUBLIC_WORKSPACE_WPS_OUTPUTS_SUBDIR
```

Comment:
@mishaschwartz 
I do not recall if I need to had `COWBIRD_PUBLIC_WORKSPACE_WPS_OUTPUTS_SUBDIR` to `EXTRA_VARS` or not when it is already in `DELAYED_EVAL`. Is it included automatically?

Reply:
No it isn't. You should add it EXTRA_VARS as well